migrations/004.do.inventory-update.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TYPE enum_material AS ENUM('Dragon Talon', 'Dragon Hide', 'Dragon Fang');
+CREATE TABLE materials (
+    type enum_material NOT NULL,
+    quantity INT,
+    PRIMARY KEY(player_id, material_id)
+);
+CREATE TYPE enum_geartype AS ENUM('DragonsBite');
+CREATE TABLE equipment (
+    id SERIAL PRIMARY KEY,
+    player_id INT NOT NULL references players(id) ON DELETE CASCADE,
+    type enum_geartype NOT NULL
+);
+CREATE TABLE equipment_mods (
+    id SERIAL PRIMARY KEY,
+    equipment_id INT NOT NULL REFERENCES equipment (id) ON DELETE CASCADE,
+    mods JSONB NOT NULL
```

Comment:
Probably good to put a default on this and enforce that it is at least an array value. No need to check format of the entries, but at least we can validate as much as we will know for sure.

```suggestion
    mods JSONB NOT NULL DEFAULT '[]' CHECK (json_typeof(mods) = 'array')
```